sdk/template/ci.yml

```diff
@@ -79,6 +79,17 @@ extends:
         safeName: azuresdktemplate
         releaseInBatch: ${{ parameters.release_azuresdktemplate }}
         skipPublishDocMs: true
+        # If you're copying template for a new pipeline, remove triggeringPaths. Template is a default library
```

Comment:
I think this set can be removed now given template will be added at the end if there are no other packages detected. There is no reason to try and maintain this list of other paths.

Reply:
Fair enough. I'll push another commit and remove these.